force-app/main/default/classes/generateCertificates.cls

```diff
@@ -0,0 +1,43 @@
+public class generateCertificates {    
+    @InvocableMethod(label='Generate certificates' description='Generate certificates for every participant of the training')
+    public static void generatePdfs(List<Id> ids) {
+        Id trainingId = ids[0];
+        List<Participant__c> participants = [SELECT Id, Training__c, Contact_Name__r.Name
+                                            FROM Participant__c
+                                            WHERE Training__c = :trainingId];
+
+        List<ContentVersion> listcv = new List<ContentVersion>();
+        List<ContentDocumentLink> listcdl = new List<ContentDocumentLink>();
+            
+        for(Participant__c particip :participants) {
```

Comment:
You can also use participant instead of particip. It is just a few characters more, but increases the readability of the code a lot.